ResearchRequirements/STRINGS.cs

```diff
@@ -96,7 +96,7 @@ public class REQUIREMENTS
             public static LocString GAS_DISTRIBUTION            = (LocString)"Have Senior Pilot with Mouth Breather trait: {0}";
             public static LocString ADVANCED_SCANNERS           = (LocString)"Scan deep space hexes (at least 3 tiles away from your asteroids): {0}";
             public static LocString SOLID_DISTRIBUTION          = (LocString)"Have Senior Pilot with Bottomless Stomach trait: {0}";
-            public static LocString NUCLEAR_RESEARCH            = (LocString)"Research 45 technologies: {0}";
+            public static LocString NUCLEAR_RESEARCH            = (LocString)"Research 50 technologies: {0}";
```

Comment:
Would this work as well?
```suggestion
            public static LocString NUCLEAR_RESEARCH            = (LocString)"Research {0} technologies: {0}";
```

Reply:
Good catch, thanks! To be consistent with other strings it should be "Research Technologies: {0}" because setting file can change the number of thechs required and I can never know how many of them will be there. Info pasted into {0} will have that numbers with correct values.

Reply:
Makes sense, I'll update the PR to match